slides/kube/helm.md

```diff
@@ -0,0 +1,205 @@
+# Managing stacks with Helm
+
+- We created our first resources with `kubectl run`, `kubectl expose` ...
+
+- We have also created resources by loading YAML files with `kubectl apply -f`
+
+- For larger stacks, managing thousands of lines of YAML is unreasonable
+
+- These YAML bundles need to be customized with variable parameters
+
+  (E.g.: number of replicas, image version to use ...)
+
+- It would be nice to have an organized, versioned collection of bundles
+
+- It would be nice to be able to upgrade/rollback these bundles carefully
+
+- Helm offers all these things!
```

Comment:
Is this the place to say Helm is an open source project and maybe link to https://helm.sh/?

Reply:
Yes! Added, thanks for the suggestion.